tool/lib/can_actions.py

```diff
@@ -15,8 +17,6 @@
 
 # Global CAN interface setting, which can be set through the -i flag to cc.py
 # The value None corresponds to the default CAN interface (typically can0)
-DEFAULT_INTERFACE = None
```

Comment:
My spontaneous reaction is that I am hesitant to move this setting away from `can_actions`. See the comment for `tool/lib/globals.py`.